src/core/server/http/logging/ecs.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+/**
+ * Typings for some ECS fields which core uses internally.
+ * These are not a complete set of ECS typings and should not
+ * be used externally; the only types included here are ones
+ * currently used in core.
+ *
+ * @internal
+ */
+export interface EcsEvent {
```

Comment:
Wasn't sure how detailed we should get with these, and whether there's yet any need to share them elsewhere in core. So for now I colocated them with the only part of core that uses them, but open to any better suggestions.

Reply:
Should we consider moving these to the logging service itself? That way, if/when there are changes to ECS all the loggers would benefit from shared definitions.
I don't think we need to do that immediately (unless someone else feels differently) but might need to in the future.

Reply:
SGTM, I'll move them to logging service so they can be re-used in the PR @TinaHeiligers is working on, and we can gradually add to them over time.